hail/python/hail/docs/experimental/index.rst

```diff
@@ -76,3 +76,10 @@ Genetics Methods
 .. autofunction:: gather
 .. autofunction:: separate
 .. autofunction:: spread
+
+`vcf_combiner`
+--------------
```

Comment:
Make sure you build and inspect the docs. I just noticed the ldscsim link above seems to be broken, which appears to be fixed by adding a header (like `Functions`) above the list of autofunctions.

Also, putting this below all the function docs makes it almost impossible to discover. How about moving the `vcf_combiner` section between `ldscsim` and the autofunctions (with a new header).